go-controller/pkg/node/gateway_init.go

```diff
@@ -105,15 +112,23 @@ func (n *OvnNode) initGateway(subnets []*net.IPNet, nodeAnnotator kube.Annotator
 		}
 	}
 
-	gatewayNextHop := net.ParseIP(config.Gateway.NextHop)
+	var gatewayNextHops []net.IP
+	// TODO:
+	//   Should config.Gateway.NextHop be a slice of nexthops?
+	//   If so, if dual-stack, but only one NextHop provided, what then?
```

Comment:
We aren't going to be using this functionality anyway. We can leave it single-stack for now. In the dual-stack case, just always use the default next hop for the IP family that doesn't have an explicitly-specified next hop